src/System.IO.FileSystem/tests/GetTime_str.cs

```diff
@@ -0,0 +1,128 @@
+// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System;
+using System.IO;
+using System.Collections;
+using System.Globalization;
+using System.Text;
+using System.Threading;
+using System.Threading.Tasks;
+using System.Runtime.CompilerServices;
+using Xunit;
+
+namespace System.IO.FileSystem.Tests
+{
+    public abstract class GetTime_str : FileSystemTest
+    {
+        private readonly string NonExistantFileName = "nonexistant_file_name";
+        private readonly TimeSpan CreateCheckGap = TimeSpan.FromMinutes(8);
+
+        protected abstract DateTime GetTime(string path);
+
+        [Fact]
+        public void FileNotFound()
+        {
+            // If the directory described in the path parameter does not exist,
+            // this method returns 12:00 midnight, January 1, 1601 A.D. (C.E.)
+            // Coordinated Universal Time (UTC), adjusted to local time.
+            DateTime time = GetTime(NonExistantFileName);
```

Comment:
IIRC, this will throw on Linux/OSX.  We should decide whether we want to match the wonky behavior Windows has (which, if I remember correctly, was mentioned in a comment as still existing only for back compat) or if we want to continue to throw.  Either way, we'd need to change something, either special-casing this test or changing the Unix implementation.